app/komponenter/banner/Banner.tsx

```diff
@@ -1,18 +1,25 @@
 import css from './banner.module.css';
 import TekstBlokk from '../tekstblokk/TekstBlokk';
+import { Heading } from '@navikt/ds-react';
 import { ETypografiTyper } from '~/typer/typografi';
-import { useTekster } from '~/hooks/contextHooks';
-import { ESanityMappe } from '~/typer/felles';
-
-const Banner: React.FC = () => {
-  const { bannerTekst } = useTekster(ESanityMappe.FELLES);
+import { ISanityDokument } from '~/typer/sanity/sanity';
 
+interface Props {
+  bannerTekst: ISanityDokument | string;
+}
+const Banner: React.FC<Props> = ({ bannerTekst }) => {
   return (
     <section className={`${css.bannerStil}`}>
-      <TekstBlokk
-        tekstblokk={bannerTekst}
-        typografi={ETypografiTyper.BANNER_HEADING}
-      />
+      {typeof bannerTekst == 'string' ? (
```

Comment:
```suggestion
      {typeof bannerTekst === 'string' ? (
```

Man bruker som regel trippel = i javascript :)

Reply:
Av nysgjerrighet. Hvorfor? 🥸
=== betyr object og verdi sammenlikning 
== betyr kun verdi sammenliking

Reply:
Nja, `==` gjør om typen til variablene som sjekkes hvis de ikke er av samme type - noe som kan føre til uante utfall. For eksempel vil `100 == "100"` gi true, `1 == true` osv. 

Idette tilfellet vil vi sjekke at strengen som returneres fra `typeof` er `string` så da gir det mening med `===`. I 99% av sjekkene våre er det nok `===` du ønsker.